### PR TITLE
Rubicon: hardcode EUR to USD for floors

### PR DESCRIPTION
**Pull request by snahornyi**

### CODE REVIEW

File: adapters/rubicon/rubicontest/exemplary/simple-video.json [hunk 1]
```diff
@@ -71,6 +71,8 @@
           "w": 1024,
           "h": 576
         },
+        "bidfloor": 1,
+        "bidfloorcur": "EuR",
```
Comment (VeronikaSolovei9) [inline adapters/rubicon/rubicontest/exemplary/simple-video.json:75]: Please add a test case for negative or zero bid floor and non "Eur" currency.

File: adapters/rubicon/rubicon.go [hunk 2]
```diff
@@ -750,6 +750,16 @@ func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *ada
 			continue
 		}
 
+		// Will be replaced after https://github.com/prebid/prebid-server/issues/1482 resolution
+		bidFloor := thisImp.BidFloor
+		bidFloorCur := thisImp.BidFloorCur
+		if bidFloor != 0 {
```
Comment (VeronikaSolovei9) [inline adapters/rubicon/rubicon.go:756]: `!=` or `> 0`?

**Review by VeronikaSolovei9 — APPROVED** [review]
Thank you for the fixes! LGTM

**Review by guscarreon — APPROVED** [review]
LGTM

**Pull request merged**